Packs/CortexXDR/ReleaseNotes/5_0_11.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex XDR Malware - Incident Enrichment
+
+Fixed an issue where the `xdr-list-risky-users` and `xdr-list-risky-hosts` commands would fail when the XDR Identity Threat Module is disabled or the license was missing.
```

Comment:
```suggestion
Fixed an issue where the ***core-list-risky-users*** and ***core-list-risky-hosts*** commands would fail when the XDR Identity Threat Module was disabled or the license was missing.
```